source/reference/minio-mc/mc-support-inspect.rst

```diff
@@ -0,0 +1,143 @@
+======================
+``mc support inspect``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support inspect
+
+Description
+-----------
+
+:mc-cmd:`mc support inspect` allows you to collect files, including metadata files, from all backend drives based on a specified path. 
+The command produces a zip file that includes all matching files with their respective *host+drive+path*. 
+
+MinIO Engineering may request this output as part of diagnostics in |subnet|.
```

Comment:
we go back and forth between `|subnet|` and `|SUBNET|`. I think its the latter, though it seems to work just fine. Should still be consistent if we need to find-replace it later.